### PR TITLE
Fix readme in doc site

### PR DESCRIPTION
**Pull request by resouer**
Sorry, my previous PR messed up the link in README (pointing to a wrong quick start), now fixed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +27,13 @@ For platform builders, KubeVela is the core engine that empowers them to create
...
 ![alt](docs/resources/how-it-works.png)
 
-## Quick Start
+## Core Concepts
+
+Learn KubeVela starting from its [core concepts](https://kubevela.io/#/en/concepts).
+
+## Installation
 
-Quick start guides are available on [this section](https://kubevela.io/#/en/quick-start).
```
Comment (wonderflow) [inline README.md:34]: the original link is right, why change quick start link to installation?

File: docs/en/README.md [hunk 2]
```diff
@@ -19,9 +17,13 @@ For platform builders, KubeVela is the core engine that empowers them to create
...
 ![alt](../resources/how-it-works.png)
 
-## Quick Start
+## Core Concepts
+
+Learn KubeVela starting from its [core concepts](en/concepts.md).
+
+## Installation
 
-Quick start tutorials are available on [this section](/en/quick-start).
```
Comment (wonderflow) [inline docs/en/README.md:24]: same here

File: docs/en/design.md [hunk 3]
```diff
@@ -1,135 +0,0 @@
-# KubeVela Design
```
Comment (wonderflow) [inline docs/en/design.md:1]: I think we can remove the design doc somewhere instead of delete it?

**Review by wonderflow — APPROVED** [review]
Generally LGTM, please make sure the link

File: README.md (hunk 1 above)
Comment (resouer) [reply README.md:34]: @wonderflow Yes, and we need to add a pure k8s quick start doc. Actually I have already created a issue for you:  https://github.com/oam-dev/kubevela/issues/1054

**Pull request merged**